app/commands/user/reputation_token/award_for_pull_request_author.rb

```diff
@@ -8,6 +8,7 @@ class AwardForPullRequestAuthor
       def call
         return unless has_author?
         return unless merged?
+        return if v3_migration_pr?
```

Comment:
I'd rather this was a generic "no-reputation" label. Otherwise we'll end up with loads of rules here.

Reply:
That makes sense. It will be a bit more work updating all the existing PRs though